app/src/main/res/layout/list_blood_sugar_history_item.xml

```diff
@@ -44,46 +44,67 @@
     android:layout_gravity="center_vertical"
     android:src="@drawable/ic_blood_sugar_outline"
     app:layout_constraintBottom_toTopOf="@id/bottomGuideline"
-    app:layout_constraintEnd_toStartOf="@id/readingsTextView"
+    app:layout_constraintEnd_toStartOf="@id/flow"
+    app:layout_constraintHorizontal_bias="0"
     app:layout_constraintStart_toEndOf="@id/startGuideline"
     app:layout_constraintTop_toBottomOf="@id/topGuideline"
     tools:ignore="ContentDescription" />
 
-  <org.simple.clinic.widgets.TintableCompoundDrawableTextView
+  <androidx.constraintlayout.helper.widget.Flow
```

Comment:
~Good choice with Flow here, but it has an error in the file which says it cannot find the widget in XML. Can you tell me what's wrong?~ Solved in next commit

Reply:
It probably happened because you need to Gradle sync the project since I have updated the dependency.

Reply:
Yes it did happen because of that.